src/crimson/osd/ops_executer.cc

```diff
@@ -566,7 +566,9 @@ OpsExecuter::do_execute_op(OSDOp& osd_op)
       return backend.zero(os, osd_op, txn, *osd_op_params, delta_stats);
     }, true);
   case CEPH_OSD_OP_SETALLOCHINT:
-    return osd_op_errorator::now();
+    return do_write_op([this, &osd_op] (auto& backend, auto& os, auto& txn) {
```

Comment:
](

Reply:
Fixed that. However, the entire switch uses the space, so – for take of consistency – we should reformat also there. I will send a follow-up PR.